test/jenkins_tests/run_multi_node_tests.sh

```diff
@@ -12,3 +12,9 @@ python $ROOT_DIR/multi_node_docker_test.py \
     --docker-image=$DOCKER_SHA \
     --num-nodes=5 \
     --test-script=/ray/test/jenkins_tests/multi_node_tests/test_0.py
+
```

Comment:
The problem with the tests in Jenkins may just be that we were starting the docker image with a small amount of shared memory (1GB), so maybe it would be sufficient to just increase that number.